.circleci/config.yml

```diff
@@ -33,31 +33,32 @@ jobs:
       - run:
           name: Build source
           command: node common/scripts/install-run-rush.js rebuild --verbose
-      - persist_to_workspace:
-          root: .
+      - save_cache:
```

Comment:
It's a bit unfortunate that the `workspace` don't work, even though they're supposed to be equivalent and get removed immediately on build finish and not persist longer.

Reply:
I tried..